test/user.test.js

```diff
@@ -0,0 +1,34 @@
+const dotenv = require('dotenv');
+const request = require('supertest');
+const { createApp } = require('../app');
+const { appDataSource } = require('../models/data-source');
+dotenv.config();
+
+describe('social-login', () => {
+  let app;
+
+  beforeAll(async () => {
+    app = createApp();
+    await appDataSource.initialize();
+  });
+  afterAll(async () => {
+    await appDataSource.destroy();
+  });
```

Comment:
모든 테스트가 종료된 이후 테스트 용 데이터베이스에 추가된 데이터를 삭제하는 코드도 추가하면 좋을 것 같습니다.

```suggestion
  afterAll(async () => {
    await appDataSource.truncate('tableName');
    await appDataSource.destroy();
  });
```